app/controllers/admin/orders_controller.rb

```diff
@@ -1,9 +1,20 @@
 # frozen_string_literal: true
 
 module Admin
-  class OrdersController < ApplicationController
+  class OrdersController < AdminController
     def index
-      render :index, locals: { orders: Order.includes(:employee).all }
+      render :index, locals: { orders: Order.includes(:employee).all.order(:status) }
+    end
+
+    def update
+      order.delivered!
```

Comment:
What if someone will somehow try to deliver already delivered order?
Also it would be good to display notice if status update for some reason will not succeed.

I would consider passing some argument (like status: :delivered) or something like this to make this action a bit more flexible, so it is updating the Order based on the input params

Reply:
I update my controller to be ready for different situations :)